network/network.go

```diff
@@ -259,6 +268,44 @@ func newNetwork(conf *Config, log *logger.SubLogger, opts []lp2p.Option) (*netwo
 	return n, nil
 }
 
+func findRelayPeers(h func() lp2phost.Host, log *logger.SubLogger) func(ctx context.Context,
+	num int) <-chan lp2ppeer.AddrInfo {
+	return func(ctx context.Context, num int) <-chan lp2ppeer.AddrInfo {
+		log.Debug("try to find relay peers", "num", num)
```

Comment:
I think it's better to use the `Info` level for logging here. what do you think?